eth/transaction.go

```diff
@@ -127,3 +127,16 @@ func (t *Transaction) MarshalJSON() ([]byte, error) {
 	u := unknown(*t)
 	return json.Marshal(&u)
 }
+
+// NewTransaction creates a new Transaction with the minimum values required for signing a transaction
+func NewTransaction(nonce uint64, gasPrice uint64, gasLimit uint64, toAddr string, value uint64, data string) (*Transaction, error) {
+	t := Transaction{
+		Nonce:    QuantityFromUInt64(nonce),
+		GasPrice: QuantityFromUInt64(gasPrice),
+		Gas:      QuantityFromUInt64(gasLimit),
+		To:       MustAddress(toAddr),
+		Value:    QuantityFromUInt64(value),
+		Input:    *MustData(data),
```

Comment:
I don't know which is best, to assume the data is "0x" prefixed or if I should append it for the user? I guess I could check for it and if not, add it for the user? 🤔

Reply:
Maybe just accept `eth.Quantity` etc values, that way you know they are already parsed correctly?

Reply:
I changed it to accept Quantity, Address, Data, etc and then realize that after those changes calling the function is just an extra function you have to call rather than just creating a tx using tx:= {} so removed the function entirely. I can add it back if anyone has any objections